### PR TITLE
feat/lt: allow passing key name and user data to lt

### PR DESCRIPTION
**Pull request by joao-fontenele**
add optional params to launch template module:

- user_data
- key_name

### CODE REVIEW

**Comment by mtulio** [comment]
@joao-fontenele that's no problem to the Ansible role keep support for `key_name`, but, for security reasons, we **don't** advise to use this in production environment.

**Comment by joao-fontenele** [comment]
@raphapr in fact, when `user_data` was the play failed. To fix it, I removed the b64 encode from inside the role

File: vars/launch-template-data.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ launch_template_data:
   EbsOptimized: "{{ item_scale.lt_spec.ebs_optimized  | d('false') | bool }}"
   Monitoring:
     Enabled: "{{ item_scale.lt_spec.instance_monitoring | d('false') | bool }}"
+  UserData: "{{ item_scale.lt_spec.user_data | d(omit) }}"
```
Comment (raphapr) [inline vars/launch-template-data.yml]: what about that?
```suggestion
  UserData: "{{ item_scale.lt_spec.user_data | d(omit) | b64encode }}"
```

Comment (raphapr) [reply vars/launch-template-data.yml]: I dont know if that's gonna work. maybe fail when b64encode is applied to nothing (omit)

Comment (joao-fontenele) [reply vars/launch-template-data.yml]: I've tried this, and the result is not omited, but rather, becomes a strange `__omit_place_holder_`

image

**Pull request merged**